tests/test_utils.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+
+from validators import ValidationFailure
+
+from fds.utils import get_git_repo_name_from_url, construct_dvc_url_from_git_url_dagshub, is_url
+
+
+class TestFds(unittest.TestCase):
+
+    def test_get_git_repo_name_from_url(self):
+        repo_name = get_git_repo_name_from_url("git@github.com:DAGsHub/fds.git")
+        assert repo_name == "fds"
+
+    def test_get_git_repo_name_from_url_from_https(self):
+        repo_name = get_git_repo_name_from_url("https://github.com/DAGsHub/fds.git")
+        assert repo_name == "fds"
+
+    def test_construct_dvc_url_from_git_url_dagshub(self):
+        repo_url = construct_dvc_url_from_git_url_dagshub("https://github.com/DAGsHub/fds.git")
+        assert repo_url == "https://github.com/DAGsHub/fds.dvc"
+
+    def test_is_url(self):
+        isUrl = is_url("https://github.com/DAGsHub/fds.git")
+        assert isUrl == True
+
+    def test_is_not_url(self):
+        isUrl = is_url("foobar")
+        assert type(isUrl) == ValidationFailure
```

Comment:
More test cases required - e.g. `s3://foobar` and other possible "URLs" like relative or absolute file paths which can point to DVC remotes.
If it's too much of a hassle to try to support all cases we can leave it for later and not support supplying a URL as a parameter - it also has other issues anyway, like not giving the user a way to define authentication

Reply:
Got it, i'll not support urls for this iteration